hw7/UI.py

```diff
@@ -0,0 +1,51 @@
+#user interface
+
+import data_prov
+import oper_file
+from data_prov import *
+from oper_file import dir_export2
+from oper_file import dir_import
+import controller
+
+
+
+# функция отображения задач
+def view(start):
+    while True:
+        print("\npress q to quit")
+        print("1: create directory" #предлагаем выбрать действие
+              " \n2: import directory"
+              " \n3: export directory"
+              " \n4: find person")
+        print()
+        size = 0
+        inp = input()
+        if inp == "1":
+            print("Input directory size:")
+            size = int(input())
```

Comment:
```suggestion
            size = int(input("Input directory size:\n))
```
Почему бы не сделать так?